src/cursor/cur_dump.c

```diff
@@ -236,7 +240,12 @@ __curdump_get_value(WT_CURSOR *cursor, ...)
     } else {
         WT_ERR(child->get_value(child, &item));
 
-        WT_ERR(__raw_to_dump(session, &item, &cursor->value, F_ISSET(cursor, WT_CURSTD_DUMP_HEX)));
+        if (F_ISSET(cursor, WT_CURSTD_DUMP_PRETTY))
+            (void)__wt_buf_set_printable_format(
```

Comment:
This should probably be `WT_IGNORE_RET_PTR()`, `(void)` doesn't always keep the compiler quiet.

Reply:
c1bf9f1